include/picongpu/param/fusion.param

```diff
@@ -0,0 +1,54 @@
+/* Copyright 2025 Filip Optolowicz
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "picongpu/param/particleFilters.param"
+#include "picongpu/particles/fusion/fusion.def"
+#include "picongpu/plugins/misc/SpeciesFilter.hpp"
+
+#include <pmacc/meta/conversion/MakeSeq.hpp>
+
+namespace picongpu
+{
+    namespace particles
+    {
+        namespace fusion
+        {
+            namespace precision
+            {
+                using float_COLL = float_64;
```

Comment:
copy past issue: please remove `float_COLL` if you not use it or rename ist e.g. to `float_fustion` or `float_Fusion`, we do not have an exact naming schema for this kind.